app/services/etherscan_service.rb

```diff
@@ -13,10 +13,30 @@ def is_proof_member?(address)
       address: address
     )
 
-    if response.dig('result')&.last&.dig('to')&.downcase == address.downcase
+    transactions = response.dig('result')
+
+    inCount = 0
+    outCount = 0
+
+    transactions.each do |child|
```

Comment:
Would you be open to changing this to `transactions.each do |transaction|`  or `transactions.each do |tx|` to make it a bit clearer what is being iterated over?